ansys/dpf/post/dpf_path.py

```diff
@@ -66,35 +66,42 @@ def __init__(self, coordinates, scoping=None):
         ...     coord_copy[1] = coord_copy[0] + i * 0.001
         ...     coordinates.append(coord_copy)
         >>> dpf_path = post.DpfPath(coordinates=coordinates)
+
         """
-        self._coordinates = coordinates
+        # check inputs
         error_text = """coordinates attribute must
         be a list of list of int."""
         if not isinstance(coordinates, list):
             raise ValueError(error_text)
         else:
             if len(coordinates) > 0 and not isinstance(coordinates[0], list):
                 raise ValueError(error_text)
-        self._scoping = scoping
-        if self._scoping is None:
-            self._scoping = Scoping(location=locations.nodal)
-            self._scoping.ids = list(range(1, len(coordinates) + 1))
+        _scoping = None
+        if scoping is None:
+            _scoping = Scoping(location=locations.nodal)
+            _scoping.ids = list(range(1, len(coordinates) + 1))
```

Comment:
@anslpa is that not len(coordinates)/3 ?

Reply:
@cbellot000 It is len(coordinates) as far as I am expecting type(coordinates)=list[list[int]], but I can add some check in case I would have type(coordinates)=list[int]